fenix/app/src/main/res/values/strings.xml

```diff
@@ -306,12 +306,16 @@
     <string name="onboarding_home_enable_notifications_negative_button">Not now</string>
 
     <!-- Juno first user onboarding flow experiment, strings are marked unused as they are only referenced by Nimbus experiments. -->
+    <!-- Title for set firefox as default browser screen used by Nimbus experiments. -->
+    <string name="juno_onboarding_default_browser_title_nimbus_2" tools:ignore="UnusedResources">We love keeping you safe</string>
```

Comment:
We also need to change this in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/onboarding.fml.yaml#14